### PR TITLE
Added CI estimation for regression parameters and its visualization

### PR DESCRIPTION
**Pull request by TomiSzucs**

### CODE REVIEW

**Review by krajcsi — CHANGES_REQUESTED** [review]
Looks promising!

File: cogstat/cogstat_stat.py [hunk 1]
```diff
@@ -531,6 +531,51 @@ def confidence_interval_t(data, ci_only=True):
 
 ### Variable pairs ###
 
+def variable_pair_regression_coefficients(slope, intercept, std_err, intercept_stderr, meas_lev, n):
+    """
+    Calculate point and interval estimates of regression parameters in a regression analysis.
+
+    Parameters
+    ----------
+    slope : Slope of the regression line
```
[the remark this answers is not in the document]
Comment (TomiSzucs) [reply cogstat/cogstat_stat.py]: Added parameter types.

File: cogstat/cogstat_chart.py [hunk 2]
```diff
@@ -637,17 +648,40 @@ def create_variable_pair_chart(data, meas_lev, slope, intercept, x, y, raw_data=
 
         if meas_lev == 'int':
             # Display the data
-            ax.scatter(xvalues, yvalues, xy_freq, color=theme_colors[0], marker='o')
+            if raw_data:
+                ax.scatter(xvalues, yvalues, xy_freq, color=theme_colors[0], marker='o')
+                # ax.scatter(data[x], data[y], color=theme_colors[0], marker='o')
```
Comment (krajcsi) [inline cogstat/cogstat_chart.py:653]:
```suggestion
                # this version in the comment would not make number pairs with multiple cases larger
                # ax.scatter(data[x], data[y], color=theme_colors[0], marker='o')
```

**Review by krajcsi — APPROVED** [review]
Looks great, thank you!

**Pull request merged**